src/components/BulkOverrideDialog/BulkOverrideInfo.js

```diff
@@ -0,0 +1,212 @@
+      : {
+        ...loans,
+        [id]: loan
+      };
+    const allChecked = size(checkedLoans) === failedRenewals.length;
+
+    this.setState({
+      checkedLoans,
+      allChecked,
+    });
+  };
+
+  isLoanChecked = (id) => {
+    const { checkedLoans } = this.state;
+
+    return (id in checkedLoans);
+  };
+
+  handleDateTimeChanged = (datetime) => {
+    if (datetime !== 'Invalid date') {
+      this.setState({ datetime });
+    }
+  };
+
+  handleAdditionalInfoChange = (event) => {
+    this.setState({ additionalInfo: event.target.value });
+  };
+
+  submitOverride = () => {
+    // Todo: should be implemented, don't have back-end yet
+  };
+
+  checkLoanPolicies() {
+    const { loanPolicies } = this.props;
+
+    for (const loanPolicyId of Object.keys(loanPolicies)) {
+      const { renewable } = find(this.loanPoliciesRecords, { id: loanPolicyId });
+
+      if (!renewable) {
+        return true;
+      }
+    }
+
+    return false;
+  }
+
+  render() {
+    const {
+      stripes,
+      failedRenewals,
+      loanPolicies,
+      requestCounts,
+      errorMessages,
+      onCancel,
+      intl: { formatMessage },
+    } = this.props;
+
+    const {
+      checkedLoans,
+      allChecked,
+      additionalInfo,
+      datetime,
+    } = this.state;
+
+    const dateSelectorDisplayed = this.checkLoanPolicies();
+    const canBeSubmittedWithDateSelector = dateSelectorDisplayed ? datetime : true;
+    const canBeSubmitted = additionalInfo && !isEmpty(checkedLoans) && canBeSubmittedWithDateSelector;
+    const selectedItems = Object.keys(checkedLoans).length;
+
+    return (
+      <div>
+        <Layout className="flex">
+          <Layout className="flex">
+            <SafeHTMLMessage
+              id="ui-users.brd.itemsSelected"
+              values={{ count: selectedItems }}
+            />
+          </Layout>
+        </Layout>
+        {
+          dateSelectorDisplayed &&
+          <DueDatePicker
+            initialValues={this.datePickerDefaults}
+            stripes={stripes}
+            dateProps={{ label: <FormattedMessage id="ui-users.cddd.dateRequired" /> }}
+            timeProps={{ label: <FormattedMessage id="ui-users.cddd.timeRequired" /> }}
+            onChange={this.handleDateTimeChanged}
+          />
+        }
+        <this.connectedLoanList
+          allChecked={allChecked}
+          stripes={stripes}
+          loanPolicies={loanPolicies}
+          requestCounts={requestCounts}
+          errorMessages={errorMessages}
+          failedRenewals={failedRenewals}
+          loanPoliciesRecords={this.loanPoliciesRecords}
+          isLoanChecked={this.isLoanChecked}
+          toggleAll={this.toggleAll}
+          toggleItem={this.toggleItem}
+        />
+        <Row>
+          <Col sm={5}>
+            <TextArea
+              label={<FormattedMessage id="ui-users.additionalInfo.label" />}
+              placeholder={formatMessage({ id: 'ui-users.additionalInfo.placeholder' })}
```

Comment:
Pleas avoid injecting intl when possible.
In this particular case you can refactor it to the following:
<FormattedMessage id="ui-users.additionalInfo.placeholder">
  {placeholder => (
    <TextArea
      label={<FormattedMessage id="ui-users.additionalInfo.label" />}
      placeholder={placeholder}
      required
      onChange={this.handleAdditionalInfoChange}
    />
  )}
</FormattedMessage>